autoPyTorch/api/base_task.py

```diff
@@ -677,6 +688,10 @@ def _do_traditional_prediction(self, num_run: int, time_left: int, func_eval_tim
                                      "Please consider increasing the run time to further improve performance.")
                 break
 
+        self._logger.debug("Run history traditional: {}".format(run_history))
+        # add run history of traditional to api run history
+        self.run_history.update(run_history.data)
```

Comment:
And then here we can create a wrapper method or something like this that calls `runhistory.add()` to update the runhistory as needed